src/py42/services/archive.py

```diff
@@ -51,53 +51,53 @@ def get_all_archives_from_value(self, id_value, id_type):
             generator: An object that iterates over :class:`py42.response.Py42Response` objects
             that each contain a page of archives.
         """
-        params = {u"{}".format(id_type): u"{}".format(id_value)}
-        return get_all_pages(self.get_page, u"archives", **params)
+        params = {f"{id_type}": f"{id_value}"}
```

Comment:
I am not sure why f-strs or .format would be needed here.

Wouldn't `parmas = {id_type: id_value}` suffice?